cdap-app-templates/cdap-etl/cdap-etl-api/src/main/java/co/cask/cdap/etl/api/Emitter.java

```diff
@@ -33,7 +33,8 @@
   void emit(T value);
 
   /**
-   * Emit an Error object.
+   * Emit an Error object. If an {@link ErrorTransform} is placed after this stage, it will be able to consume
+   * the errors. Otherwise the errors are simply dropped.
```

Comment:
it isn't, but I'm not sure how to change this without breaking backwards compatibility. Open to suggestions.

From a practical standpoint, it doesn't matter for any of our plugins since they all operate on StructuredRecord